utils/kata-manager.sh

```diff
@@ -145,6 +145,7 @@ github_get_release_file_url()
 
 	local arch
 	arch=$(uname -m)
+	[ "$arch" = "x86_64" ] && arch="amd64"
```

Comment:
Out of curiosity, James, `kata-manager` is not aware of other arches, is it?
The reason I'm asking is because we most likely would have to do the same trick for aarch64 -> arm64.

Reply:
> Out of curiosity, James, `kata-manager` is not aware of other arches, is it?

Correct. It would be up to the other architecture maintainers to add the (very minor) tweaks necessary for other architectures. And to test on those architectures of course :smile:

Reply:
Hmm.  Okay.  I personally think that knowing it'd be needed for ARM adding the aarch64 -> arm64 switch is very cheap and could be done on this PR.  But I won't block it if you don't fancy addressing it.